content/kit/pi/update-file.md

```diff
@@ -0,0 +1,27 @@
+---
+title: The update file
+---
+
+The update file is the simplest way to update your Pi with new software from us.  
+
+{{% notice tip %}}
+Using this file is the recommended way to update your kit.
+{{% /notice %}}
+
+## Using the update file
+
+1. Put the `update.tar.xz` file on a USB drive.
+2. Connect the USB drive to your robot.
+3. Turn on your robot
+
+After the update is completed, the Pi will reboot. If you have a `main.py` file on the USB as well, this will now execute, because the `update.tar.xz` file is deleted when the update begins.
+
+{{% notice warning %}}
+Do not turn off your Pi during the update process. It can take a few minutes to finish.
```

Comment:
how will they know when the update is complete?

Reply:
The Pi reboots, as said above.

Reply:
This feels rather disjoint. Could we add a step 4. to the list which is "wait for the pi to reboot"? That way it's a clear part of the main instructions that waiting is involved and that they need to be watching the pi during the update.

We could even go as far as suggesting that they include a `main.py` which contains:
``` python
import time
from robot import Robot
r = Robot()
while True:
    r.power_board.buzz(1, note='c')
    time.sleep(1.5)
```

Reply:
I was thinking of something like that. I would actually rather bake the beeping into `runusb` in that case, so they don't have to include anything on their end besides the single file.

Reply:
Either way we end up needing a power board being attached (see also above), which I'm not sure is something we really want to require.